packages/quick_actions/quick_actions/ios/Classes/FLTQuickActionsPlugin.m

```diff
@@ -50,12 +52,36 @@ - (BOOL)application:(UIApplication *)application
     performActionForShortcutItem:(UIApplicationShortcutItem *)shortcutItem
                completionHandler:(void (^)(BOOL succeeded))completionHandler
     API_AVAILABLE(ios(9.0)) {
-  [self.channel invokeMethod:@"launch" arguments:shortcutItem.type];
+  [self _handleShortcut:shortcutItem.type];
   return YES;
 }
 
+- (BOOL)application:(UIApplication *)application
+    didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  if (@available(iOS 9.0, *)) {
+    UIApplicationShortcutItem *shortcutItem =
+        launchOptions[UIApplicationLaunchOptionsShortcutItemKey];
+    if (shortcutItem) {
+      shortcutType = shortcutItem.type;
+      return NO;
+    }
+  }
+  return YES;
+}
+
+- (void)applicationDidBecomeActive:(UIApplication *)application {
+  if (shortcutType) {
+    [self _handleShortcut:shortcutType];
+    shortcutType = nil;
+  }
+}
+
 #pragma mark Private functions
 
+- (void)_handleShortcut:(NSString *)shortcut {
```

Comment:
nits: OBJC method should not start with `handleShortcut` is fine.

Reply:
done